coding-concepts.js

```diff
@@ -70,4 +98,7 @@ var learnStudent = new Learn("George")
 // console.log(learnStudent)
 
 // a) Your answer:
```

Comment:
So classes are the building blocks for objects. The learnStudent variable is creating a new instance. So looking inside the constructor, we see that we have a parameter of name that is waiting to be assigned a value (argument). Since cohort and year are hard-coded, each new object will always receive the key-value pairs of cohort: "Alpha" and year: "2022". So when passing the argument "George", it will be assigned to name.

Learn { student: 'George', cohort: 'Alpha', year: 2022 }